scripts/router-dashboard-api-wrapper.py

```
@@ -0,0 +1,298 @@
+    if FAIL2BAN_STATUS_FILE:
+        try:
+            if os.path.exists(FAIL2BAN_STATUS_FILE):
+                with open(FAIL2BAN_STATUS_FILE, "r", encoding="utf-8") as handle:
+                    payload = json.load(handle)
+                self.send_json(payload)
+                return
+            else:
+                print(f"Fail2ban snapshot file NOT FOUND at: {FAIL2BAN_STATUS_FILE}")
+        except FileNotFoundError:
+            pass
+        except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)